xcode/Subconscious/Shared/Models/SaveState.swift

```diff
@@ -9,8 +9,7 @@ import Foundation
 
 /// A simple enum representing saved state, including in-progress write.
 enum SaveState: Hashable, Equatable {
-    case draft
-    case modified
+    case unsaved
```

Comment:
Clarifying save state. Draft vs modified was a distinction without a difference in the logic. What we need to know is whether it is unsaved.